Dymension-League/frontend
# LD: Add place bet page

Fixes https://github.com/Dymension-League/frontend/issues/11

### Proposal

Create bet page is not ready yet. This PR adds support for betting.

NOTE: There are also some UI fixes and minor improvements:
1. `CreateTeamPage` now uses `initialized` variable to avoid "infinite" call on initial loading functions.
2. Add a proper local environment. We now rely on:

```
REACT_APP_LOCAL_COSMOSHIPS_ADDRESS=<local address deployed on anvil>
REACT_APP_LOCAL_GAMELEAGUE_ADDRESS=<local address deployed on anvil>   
```
More info can be found in https://github.com/Dymension-League/contracts/pull/8

### Testing

1. Mint some ships
2. Create a new team
3. Navigate to bet page.
4. Ensure that your wallet is not connected
5. Ensure that you see "No teams available for betting" message
6. Connect your wallet.
7. Ensure that you see the teams
8. Ensure that whatever you do the team is not selectable
9. Press ` 1. Initialize League TEST`
10. Ensure that whatever you do the team is not selectable
11. Press `2 End enrollment`
12. After enrollment, ensure that you can select a team.
13. After adding a bidding price, ensure that place bet button is enabled.
14. Press `Place bet` button
15. Ensure that you can place a bet, and you see a successfull message
